backend/src/Response/Admin/CaptainAmountFromOrderCash/CaptainAmountFromOrderCashResponse.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Response\Admin\CaptainAmountFromOrderCash;
+
+use OpenApi\Annotations as OA;
+
+class CaptainAmountFromOrderCashResponse
+{
+   public int $id;
+
+   public string $captainName;
+
+   public int $orderId;
+
+   public float $amount;
+
+   public int $flag;
+     /**
+     * @OA\Property(type="object", property="createdAt")
+     */
+   public object $createdAt;
```

Comment:
here we can use DateTime as a var type instead of object and avoiding using openAPI annotation.